opm/output/eclipse/Summary.cpp

```diff
@@ -1075,6 +1122,59 @@ Summary::Summary( const EclipseState& st,
     }
 }
 
+std::vector< std::pair< std::string, double > >
+efficiency_factors( const smspec_node_type* type,
```

Comment:
The elemenents in the `std::pair`are all well names; call the function something with well?

Reply:
The efficiency factors in this vector are the product of WEFACs and GEFACs. Perhaps calling the function something with well may lead the reader to believe that these are only WEFACs?

Reply:
In the rest of opm-parser function names use camelCase(), so this should too (noticed randomly just now, sorry for late comment).

Reply:
Well - that is true; but all the factors are *applied* to wells; so I would still say well in the name makes sense.